daemon/src/lib.rs

```diff
@@ -414,15 +484,31 @@ pub enum Environment {
     Test,
     Legacy,
     Unknown,
+    Maker,
```

Comment:
❓ What is a `Maker` environment?

Reply:
The more high level question is: What should the maker specify? 
To some degree the maker's environment is irrelevant and should just not be sent. But this is specific to our current setup. This might be useful for something once there are multiple makers. Not sure. If you have a better idea let me know; I wanted to overcome the decision for now and just introduced this variant.